src/documents/tutorial-gettingStartedWithInfusion/EventedComponents.md

```diff
@@ -9,23 +9,24 @@ Part of the [Getting Started with Infusion Tutorial](GettingStartedWithInfusion.
 
 ---
 
-Many times, you will be creating a component that works in an environment where other things are operating, and it will probably need to notify those other things of key **events** in its lifecycle. Events can be used to trigger changes in the visual appearance of a component, or actions by other components. For example:
+Many times, you will be creating a component that works in an environment where other things are operating, and it will probably need to notify those other things of key **events** in its lifecycle. 
```

Comment:
This entire page is no longer appropriate: We don't have a specific "evented" component as one of the types of components to choose from. Any information here that we want to keep in this tutorial should be moved to another page, and the other page that mention evented components and/or link to this page should be updated (the getting started page, the 'pick a component type' page...)